src/Main/App.js

```diff
@@ -85,6 +85,7 @@ class App extends Component {
 
     this.handleReportSelecterSubmit = this.handleReportSelecterSubmit.bind(this);
     this.handleRefresh = this.handleRefresh.bind(this);
+    this.renderContent = this.renderContent.bind(this);
```

Comment:
This is only necessary for event handlers that you bind to elements, it can be removed.